App/SB/components/FeedItem/index.tsx

```diff
@@ -60,11 +63,10 @@ const FeedItem = (props: FeedItemProps) => {
         </View>
       </View>
       <View style={styles.textContainer}>
-        <Text style={styles.text}>{payload.feed}</Text>
+        <Text style={styles.text}>{feedText}</Text>
```

Comment:
So in this case, we only want to render this `Text` element if `feedText` is  defined and has a length > 0